process/lifecycle.md

```diff
@@ -24,7 +24,7 @@ To be accepted at the Incubation stage, a project must:
 * Submit a completed [Project Contribution Proposal Template](proposal_template.md) to the TAC, or the TAC’s designated recipient for contribution proposals.
 * Provide such additional information as the TAC may reasonably request.
 * Be available to present to the TAC with respect to the project’s proposal and inclusion in ASWF. Project teams should be prepared to present a detailed (20-30 minutes in length) overview on the project in addition to speaking to the information contained in the project contribution proposal.
-* Be deemed by the TAC to add value to the mission of ASWF.
+* Be deemed by the TAC to add potential or value to the mission of ASWF.
```

Comment:
Should this be "potential value or value" instead of "potential or value"

Reply:
potential value is an improvement -- thanks.